ui/ChaosKit/OutlinedButton.qml

```diff
@@ -0,0 +1,35 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Controls.impl 2.12
+import ChaosKit 1.0
+
+Button {
+  id: outlinedButton
+  flat: true
+
+  icon.color: !outlinedButton.enabled
+      ? Theme.onSurfaceDisabled
+      : Theme.primaryColor
+
+  contentItem: IconLabel {
```

Comment:
I think you don't have to override the `contentItem`, as the settings are the same as in our Button.